test/jdk/javax/swing/JTabbedPane/4209065/bug4209065.java

```diff
@@ -49,9 +52,14 @@ private void createTabbedPane() {
         JTabbedPane tp = new JTabbedPane();
         getContentPane().add(tp);
         String text = "<html><center>If the style of the text on the tabs matches"
-                      + "<br>the descriptions, press <em><b>PASS</b></em></center></html>";
-        tp.addTab("<html><center><font size=+3>big</font></center></html>", new JLabel(text));
-        tp.addTab("<html><center><font color=red>red</font></center></html>", new JLabel(text));
-        tp.addTab("<html><center><em><b>Bold Italic!</b></em></center></html>", new JLabel(text));
+                      + "<br>the descriptions, press <em><b>PASS</b></em><br>"
+                      + "<br> NOTE: the text (tab title) may not always "
+                      + "fit the tab.</center></html>";
+        tp.addTab("<html><center><font size=+2>big</font></center></html>",
+                new JLabel(text));
```

Comment:
If you are saying text may not fit the tab, then I guess no need to change font size from +3 to +2, leave it as it is..

Reply:
@prsadhuk Font-size of +2 is where it starts to get clipped but as you suggested we can change it back to +3 because of the included note. Updated the test case.